yb-voyager/cmd/live_migration_integration_test.go

```diff
@@ -0,0 +1,711 @@
+
+	// Create a temporary export directory.
+	exportDir = testutils.CreateTempExportDir()
+	defer testutils.RemoveTempExportDir(exportDir)
+
+	createSchemaSQL := `CREATE SCHEMA IF NOT EXISTS test_schema;`
+	createTableSQL := `
+CREATE TABLE test_schema.test_live (
+	id int GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	name TEXT,
+	email TEXT,
+	description TEXT
+);`
+	insertDataSQL := `
+INSERT INTO test_schema.test_live (name, email, description)
+SELECT
+	md5(random()::text),                                      -- name
+	md5(random()::text) || '@example.com',                    -- email
+	repeat(md5(random()::text), 10)                           -- description (~320 chars)
+FROM generate_series(1, 20);`
+	dropSchemaSQL := `DROP SCHEMA IF EXISTS test_schema CASCADE;`
+
+	// Start Postgres container with live migration
+	postgresContainer := testcontainers.NewTestContainer("postgresql", &testcontainers.ContainerConfig{
+		ForLive: true,
+	})
+	if err := postgresContainer.Start(ctx); err != nil {
+		utils.ErrExit("Failed to start Postgres container: %v", err)
+	}
+
+	// Start YugabyteDB container.
+	yugabytedbContainer := testcontainers.NewTestContainer("yugabytedb", nil)
+	if err := yugabytedbContainer.Start(ctx); err != nil {
+		utils.ErrExit("Failed to start YugabyteDB container: %v", err)
+	}
+	postgresContainer.ExecuteSqls([]string{
+		createSchemaSQL,
+		createTableSQL,
+		insertDataSQL,
+	}...)
+
+	yugabytedbContainer.ExecuteSqls([]string{
+		createSchemaSQL,
+		createTableSQL,
+	}...)
+
+	defer postgresContainer.ExecuteSqls(dropSchemaSQL)
+	defer func() {
+		for i := 0; i < 5; i++ {
+			ybConn, err := yugabytedbContainer.GetConnection()
+			testutils.FatalIfError(t, err, "error while getting connection")
+			_, err = ybConn.Exec(dropSchemaSQL)
+			if err != nil {
+				if !strings.Contains(err.Error(), "cannot remove a key column") {
+					testutils.FatalIfError(t, err, "error while dropping schema")
+				}
+			} else {
+				break
+			}
+		}
+	}()
+
+	err := testutils.NewVoyagerCommandRunner(postgresContainer, "export data", []string{
+		"--export-dir", exportDir,
+		"--source-db-schema", "test_schema",
+		"--disable-pb", "true",
+		"--export-type", SNAPSHOT_AND_CHANGES,
+		"--yes",
+	}, nil, true).Run()
+	testutils.FatalIfError(t, err, "Export command failed")
+
+	start := time.Now()
+	for time.Since(start) < (30 * time.Second) { // 30 seconds timeout
+		//check for status until its streaming
+		if isExportDataStreamingStarted(t) {
+			break
+		}
+		time.Sleep(1 * time.Second)
+	}
+
+	importCmd := testutils.NewVoyagerCommandRunner(yugabytedbContainer, "import data", []string{
+		"--export-dir", exportDir,
+		"--disable-pb", "true",
+		"--yes",
+	}, nil, true)
+	err = importCmd.Run()
+	testutils.FatalIfError(t, err, "Import command failed")
+
+	time.Sleep(5 * time.Second)
+
+	start = time.Now()
+	for time.Since(start) < (30 * time.Second) { // 30 seconds timeout
+		if snapshotPhaseCompleted(t, postgresContainer.GetConfig().Password,
```

Comment:
is it possible to use our existing main code function for checking snapshot streaming phases..